content/04.results.md

```diff
@@ -2,42 +2,54 @@
 
 ### Generative Model Using Randomly Sampled Label Functions
 ![
-Grid of AUROC scores for each generative model trained on randomly sampled label functions.
-The rows depict the relationship each model is trying to predict and the columns are the edge type specific sources from which each label function is sampled.
-The right most column consists of pooling every relationship specific label function and proceeding as above.
-](https://raw.githubusercontent.com/danich1/snorkeling/ee638b4e45717a86f54a2744a813baaa90bc6b84/figures/label_sampling_experiment/transfer_test_set_auroc.png){#fig:auroc_gen_model_performance}
+Plot of AUROC scores for each generative model trained on randomly sampled label functions.
+The headers depict the relationship each model is trying to predict and the colored points are the edge type specific sources from which each label function is sampled.
+The 0th point represents our baseline model which only uses only distant supervision label functions.
+](https://raw.githubusercontent.com/danich1/snorkeling/86037d185a299a1f6dd4dd68605073849c72af6f/figures/label_sampling_experiment/transfer_test_set_auroc.png){#fig:auroc_gen_model_test_set}
+
+![
+Plot of AUPR scores for each generative model trained on randomly sampled label functions.
+The headers depict the relationship each model is trying to predict and the colored points represent the edge type specific sources from which each label function is sampled.
+The 0th point represents our baseline model which only uses distant supervision label functions.
+](https://raw.githubusercontent.com/danich1/snorkeling/86037d185a299a1f6dd4dd68605073849c72af6f/figures/label_sampling_experiment/transfer_test_set_aupr.png){#fig:aupr_gen_model_test_set}
 
 We added randomly sampled label functions to a baseline for each edge type to evaluate the feasibility of label function re-use.
```

Comment:
First sentence: what is this experiment testing. Need the context here. The figures should come after the first paragraph where they are discussed, not before.